src/main/java/Server/MopeLSPServer.java

```diff
@@ -23,21 +29,57 @@
     private DiagnosticHandler diagnosticHandler;
     private static ICompilerAdapter compiler;
     private ConfigObject cfg;
+    private CompletableFuture<Object> shut;
+    private ServerSocket socket;
+    private String path;
```

Comment:
Why are these ClassMembers here?
The MopeServerLauncher already implements and uses them to store the Config ...